client/src/locales/fr-FR.json

```
@@ -839,8 +840,9 @@
             "download": "Télécharger",
             "copyLink": "Copier le lien",
             "history": "Historique",
-            "open": "Ouvrir",
-            "details": "Détails"
+            "preview": "Aperçu",
+            "details": "Détails",
+            "edit": "Modifier"
```

To be consistent with the other menus
```suggestion
            "edit": "Éditer"
```